sass/_CardActionBar.scss

```diff
@@ -0,0 +1,41 @@
+.CoveoCardActionBar {
```

Comment:
Good idea! I just implemented something, tell me what you think. There is now an option to open the ActionBar on hover. I made it optional, but enabled by default.